Makefile

```diff
@@ -152,6 +152,15 @@ else
 endif
 endif
 
+i18n-extract-server:
+	goi18n extract -format json -outdir assets/i18n/ server/
+	touch assets/i18n/translate.es.json
```

Comment:
Is this really needed?

Reply:
This is part of my problem of not knowing well the `goi18n` tool. I going to read some documentation and try to figure out what is the best workflow there.

Reply:
Yes, it is needed to generate the translations, is everything encapsulated in the makefile.

Reply:
Awesome :+1: